project.clj

```diff
@@ -1,17 +1,20 @@
-(defproject mx.cider/enrich-classpath "1.6.2"
+(defproject mx.cider/enrich-classpath "1.7.0"
   :description "Makes available .jars with Java sources and javadocs for a given project."
 
   :url "https://github.com/clojure-emacs/enrich-classpath"
 
   :license {:name "EPL-2.0"
             :url  "https://www.eclipse.org/legal/epl-2.0/"}
 
-  :dependencies [[org.clojure/clojure "1.10.3"] ;; Hard-require a recent-enough version of Clojure, since other plugins may require an overly old one which would make Fipp fail.
-                 ^:inline-dep [fipp "0.6.25" :exclusions [org.clojure/clojure]]]
+  :dependencies [;; lein assumes it but t.deps requires it:
+                 [clj-commons/pomegranate "1.2.1"]
+                 ^:inline-dep [fipp "0.6.25" :exclusions [org.clojure/clojure]]
+                 [org.clojure/clojure "1.10.3"] ;; Hard-require a recent-enough version of Clojure, since other plugins may require an overly old one which would make Fipp fail.
+                 [org.clojure/tools.deps.alpha "0.12.1109"]]
```

Comment:
Maybe create a subproject for CLI projects only, this way we don't add t.deps as a dep for Lein consumers

Reply:
(Same for pomegranate)